agent/docker-compose.yml

```diff
@@ -8,3 +8,5 @@ agent:
     - KONTENA_PEER_INTERFACE=eth1
   volumes:
     - /var/run/docker.sock:/var/run/docker.sock
+  labels:
+    - io.kontena.container.skip_logs=1
```

Comment:
Yes, you need this if you do any debug logging while sending container logs.... so, having this label during development lets you do that.

However, there's a problem: if we start logging more debug messages, and you then enable agent `DEBUG` in production, without this label... it's going to explode :bomb: